CMakeLists.txt

```diff
@@ -40,6 +40,12 @@ ENDIF()
 
 set (WORLD_BUILDER_SOURCE_DIR ${PROJECT_SOURCE_DIR})
 
+# define some config options:
+set(WB_ENABLE_APPS ON CACHE BOOL "If enabled, applications gwb-grid and gwb-dat are built.")
```

Comment:
The apps are used in the tester, so if this is off and the testers are on, it needs to only run the tests which do not require either gwb-grid or gwb-dat. Can you add a github tester which turns off the apps, but leaves the tester on?

Reply:
For the test, please put it in test.yml and set `needs: [test_indentation,coverage]`. If you want me to add the tester, let me know, I am happy to do it.